### DISCUSSION TITLE
cucumber feature files not working

### DISCUSSION THREAD

**Discussion by nidhalBougatf** [cypress-io/cypress#15599] [Questions and Help]
Hey all,
I am having some issues while running the *.feature files. Actually, it's always showing this error :
login.feature
![loginfeature file](https://user-images.githubusercontent.com/22817092/111915816-60d1c180-8a78-11eb-8ec2-821f3ef0774d.PNG)

![feture file error](https://user-images.githubusercontent.com/22817092/111915561-3f240a80-8a77-11eb-832a-6baeda41df03.PNG)

and when running the login.js file 
![loginjs](https://user-images.githubusercontent.com/22817092/111915806-5b747700-8a78-11eb-8d61-ae9af351cdba.PNG)

![error js file](https://user-images.githubusercontent.com/22817092/111915570-48ad7280-8a77-11eb-9031-0085a0d1dff7.PNG)

By the way, I have a js file, written only in cypress without Gherkin language, that is running correctly.
test.js
![testjs](https://user-images.githubusercontent.com/22817092/111915785-48fa3d80-8a78-11eb-99dc-6a179ff45053.PNG)
Running screenshot
![working other js file](https://user-images.githubusercontent.com/22817092/111915646-975b0c80-8a77-11eb-84c8-2cc9aa2002a9.PNG)

This is my configuration files :
Cypress.json
![cypressjson](https://user-images.githubusercontent.com/22817092/111915758-2ec05f80-8a78-11eb-9173-90fa57f71fb9.PNG)

index.js
![indexjs file](https://user-images.githubusercontent.com/22817092/111915767-34b64080-8a78-11eb-8b74-c91c533f25f7.PNG)

package.json
![packgeJSON part1](https://user-images.githubusercontent.com/22817092/111915774-3bdd4e80-8a78-11eb-9b47-4babbc62c4ba.PNG)
![packgeJSON part2](https://user-images.githubusercontent.com/22817092/111915776-3da71200-8a78-11eb-9db7-f94134f18cfb.PNG)
 

Things that I've tried :
- updating cypress to 6.8.0
- updating cucumber to 4.0.1
- updating typescript +4
- clear cache of cypress

**Comment by MatiDiez** [comment]
Tried removing the "import" statement for your login.js?

**Reply by nidhalBougatf** [reply]
@MatiDiez  Yes, still same error
